diff --git a/client/src/components/ui/TabAccepted.js b/client/src/components/ui/TabAccepted.js
index 89e3b70..e691b18 100644
--- a/client/src/components/ui/TabAccepted.js
+++ b/client/src/components/ui/TabAccepted.js
@@ -214,9 +214,12 @@ export class TabAccepted extends Component {
             <div className="Tabssuggestion_main_body12">
                 <div className="Tabs_main_body1 h6fnhWdeg-tab-content">
                     <div className="Accepted_tabs_filter h6fnhWdeg-tab-controls-container">
-                        <Filter onChange={this.handleFilterUpdate} showSort={true}  page={this.state.page} count={this.state.count} onFilterChange={this.handlePaginationUpdate}/>
-                        <button className="btn btn-primary  backgroundColorBlue  h6fnhWdeg-accept-all borderRadius0">Accept All</button>
-                        <button className="btn btn-default h6fnhWdeg-reject-all borderRadius0" onClick={this.rejectAll}>Reject All</button>
+                        <Filter onChange={this.handleFilterUpdate} showSort={true} page={this.state.page} count={this.state.count} onFilterChange={this.handlePaginationUpdate} />
+                        {publications.paginatedPublications.length ?
+                            <React.Fragment>
+                                <button className="btn btn-primary  backgroundColorBlue  h6fnhWdeg-accept-all borderRadius0">Accept All</button>
+                                <button className="btn btn-default h6fnhWdeg-reject-all borderRadius0" onClick={this.rejectAll}>Reject All</button>
+                            </React.Fragment> : null}
                     </div>
                     {/* <p className="suggested_filter_text">Not finding what you're looking for? <a onClick={() => { this.props.tabClickHandler("Add Publication"); }}>Search PubMed...</a></p> */}
                     {/* <Pagination total={publications.filteredPublications.length} page={this.state.page} count={this.state.count} onChange={this.handlePaginationUpdate} /> */}
diff --git a/client/src/components/ui/TabRejected.js b/client/src/components/ui/TabRejected.js
index 863c9d3..89f038e 100644
--- a/client/src/components/ui/TabRejected.js
+++ b/client/src/components/ui/TabRejected.js
@@ -215,8 +215,10 @@ export class TabRejected extends Component {
             <div className="Tabs_main_body1 h6fnhWdeg-tab-content">
                 <div className="Rejected_tabs_filter h6fnhWdeg-tab-controls-container">
                     <Filter onChange={this.handleFilterUpdate} showSort={true} page={this.state.page} count={this.state.count} onFilterChange={this.handlePaginationUpdate}/>
+                    {publications.paginatedPublications.length ? <React.Fragment>
                     <button className="btn btn-primary backgroundColorBlue h6fnhWdeg-accept-all borderRadius0" onClick={this.acceptAll}>Accept All</button>
                     <button className="btn btn-default h6fnhWdeg-reject-all borderRadius0">Reject All</button>
+                    </React.Fragment> : null}
                 </div>
                 {/* <Pagination total={publications.filteredPublications.length} page={this.state.page} count={this.state.count} onChange={this.handlePaginationUpdate} /> */}
                 <div className="tabs_responsive_table col-md-12">
diff --git a/client/src/components/ui/TabSuggested.js b/client/src/components/ui/TabSuggested.js
index 43fa234..046d2e5 100644
--- a/client/src/components/ui/TabSuggested.js
+++ b/client/src/components/ui/TabSuggested.js
@@ -248,15 +248,16 @@ export class TabSuggested extends Component {
             <div className="Tabssuggestion_main_body12">
                 <div className="Tabs_main_body1 h6fnhWdeg-tab-content">
                     <div className="suggested_tabs_filter h6fnhWdeg-tab-controls-container">
-                        <Filter onChange={this.handleFilterUpdate} showSort={true}  page={this.state.page} count={this.state.count} onFilterChange={this.handlePaginationUpdate}/>
-                        <button
+                        <Filter onChange={this.handleFilterUpdate} showSort={true} page={this.state.page} count={this.state.count} onFilterChange={this.handlePaginationUpdate} />
+                        {publications.paginatedPublications.length ? <React.Fragment><button
                             className="btn btn-primary h6fnhWdeg-accept-all backgroundColorBlue borderRadius0"
                             onClick={this.acceptAll}
                         >Accept All</button>
-                        <button
-                            className="btn btn-primary h6fnhWdeg-reject-all colorBlue backgroundColorWhite borderRadius0"
-                            onClick={this.rejectAll}
-                        >Reject All</button>
+                            <button
+                                className="btn btn-primary h6fnhWdeg-reject-all colorBlue backgroundColorWhite borderRadius0"
+                                onClick={this.rejectAll}
+                            >Reject All</button></React.Fragment>: null}
+
                     </div>
                     {/* <Pagination total={publications.filteredPublications.length} page={this.state.page} count={this.state.count} onChange={this.handlePaginationUpdate} /> */}
                     <div className="tabs_responsive_table col-md-12">
